include/vara/Feature/FeatureModel.h

```diff
@@ -21,17 +21,22 @@ class FeatureModel {
 
 private:
   string Name;
+  string Path;
```

Comment:
```suggestion
  std::filesystem::path RootPath;
```
This makes it more explicit what the intend behind the path is.
PS: we could also use `std::filesystem::path` for paths, making the path platform independent and giving us methods to interact with the path.